/D435i_1/depth/color/points:
  extrinsic_T: [-0.3777585680065469, -0.02113807867857562937, -0.1741453055668508]
  extrinsic_R: [0.999953862994609, 4.206138519935441e-05, -0.00960573333997171, 4.206138519935441e-05, 0.9999616542055582, 0.008757188437147484, 0.00960573333997171, -0.008757188437147484, 0.9999155172001671]
/D435i_2/depth/color/points:
  extrinsic_T: [-0.1250494799208164, -0.0201369057277589838, -0.1715464286536905]
  extrinsic_R: [0.9999991145852068, 7.992852547039754e-06, -0.001330700911806117, 7.992852547039754e-06, 0.9999278465953677, 0.01201255756552353, 0.001330700911806117, -0.01201255756552353, 0.9999269611805746]
/D435i_3/depth/color/points:
  extrinsic_T: [0.125494799208164, -0.0144807867857562937, -0.1698090367521044]
  extrinsic_R: [0.9999476214343981, -8.820044773516482e-06, -0.01023495529528585, -8.820044773516482e-06, 0.9999985147896108, -0.001723467661284435, 0.01023495529528585, 0.001723467661284435, 0.9999461362240089]
/D435i_4/depth/color/points:
  extrinsic_T: [0.375094799208164, -0.02113807867857562937, -0.1675342471677332]
  extrinsic_R: [0.9999983593355689, 3.639937644973648e-06, -0.001811439461078613, 3.639937644973648e-06, 0.9999919245241082, 0.004018814915000044, 0.001811439461078613, -0.004018814915000044, 0.9999902838596773]